### ISSUE TITLE
Add documentation on how to detect the FrankenPHP version and the Caddy version underneath

### ISSUE THREAD

**Issue by Marian-Kechlibar** [dunglas/frankenphp#1237]
### Describe you feature request

I would like to display in my app's About page what is the precise version of FrankenPHP and Caddy running underneath. I am using one of the official Docker images as a base.

The best would be to describe in the documentation, which environment variables or functions can be used to obtain this version information.

**Comment by withinboredom** [comment]
See #1225

**Comment by samdark** [comment]
I don't think it exposes this info. There's `SERVER_SOFTWARE` but it says simply "FrankenPHP". So the following should do the job:

```php
$frankenString = exec('frankenphp -v');
// in my case it is:
// FrankenPHP (WolfiPHP) 1.3.6 Caddy v2.8.4 => /usr/src/caddy@(devel)
```

**Comment by withinboredom** [comment]
Correct. It currently doesn't expose the version but the issue I linked above is also linked to an open PR to expose that version string.

**Comment by henderkes** [comment]
phpversion('frankenphp'); retur s the frankenphp version
